package.json

```diff
@@ -92,7 +92,7 @@
     "build": "pnpm build-package _internal && pnpm build-package core && pnpm build-package infinite && pnpm build-package immutable && pnpm build-package mutation && pnpm build-package subscription",
     "build:e2e": "pnpm next build e2e/site",
     "build-package": "bunchee --cwd",
-    "attw": "attw --pack",
+    "attw": "attw --pack --ignore-rules cjs-only-exports-default",
```

Comment:
why we need to ignore this rule?

can we also run attw in the test:build process on ci so it's get covered on CI?

Reply:
It has already been included in CI.  

We need to ignore this rule because SWR use default exports which could cause problems. https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/CJSOnlyExportsDefault.md#-cjs-default-export